frontend/src/pages/ActivityReport/Pages/Review/Submitter/index.js

```diff
@@ -31,6 +31,13 @@ const Submitter = ({
   const submitted = status === REPORT_STATUSES.SUBMITTED;
   const needsAction = status === REPORT_STATUSES.NEEDS_ACTION;
   const approved = status === REPORT_STATUSES.APPROVED;
+  const [approverStatusList, updateApproverStatusList] = useState([]);
+
+  useEffect(() => {
+    if (formData && approvingManager && status) {
+      updateApproverStatusList([{ approver: approvingManager.fullName, status }]);
+    }
+  }, [formData, approvingManager, status]);
```

Comment:
I can see why we need the `useEffect` here, a draft report doesn't have an approving manager yet (but can specify one). I could go either way keeping the `useEffect` for the approver for consistency vs removing it for the approver.